src/control/server/ctl_storage_rpc_test.go

```diff
@@ -546,6 +547,73 @@ func TestServer_CtlSvc_StorageScan_PostIOStart(t *testing.T) {
 				Scm: &ScanScmResp{State: new(ResponseState)},
 			},
 		},
+		"scan twice bdev meta with multiple io servers up": {
```

Comment:
Not totally following what "scan twice" is testing. I would guess the use of the cache, but did something change between the two scans that would make it obvious that we're using the cache, not making the second call? If there's anything you can do to clarify that, I think it would be helpful for future developers (and ourselves in the future).

Reply:
there was a bug that added duplicate SMD entries to the cache with multiple scans, this test checks cache contents is not duplicated over multiple scans, added comment in follow-up https://github.com/daos-stack/daos/pull/3528